sass/chosen.scss

```diff
@@ -5,140 +5,130 @@
+$chosen-border: #aaa;
+$chosen-highlight: #3875d7;
+$chosen-text: #444;
+
 /* @group Base */
 .chosen-container {
-  position: relative;
   display: inline-block;
-  vertical-align: middle;
+  font-family: sans-serif;
   font-size: 13px;
   @include user-select(none);
+  position: relative;
+  vertical-align: middle;
   * {
     @include box-sizing(border-box);
   }
   .chosen-drop {
+    background: $chosen-background;
+    border: 1px solid $chosen-border;
+    border-radius: $chosen-radius;
+    box-shadow: 0 2px 8px rgba(#000, 0.2);
+    left: -9999px;
+    margin-top: 2px;
+    overflow: hidden;
+    padding: 5px;
     position: absolute;
     top: 100%;
-    left: -9999px;
-    z-index: 1010;
     width: 100%;
-    border: 1px solid #aaa;
-    border-top: 0;
-    background: #fff;
-    box-shadow: 0 4px 5px rgba(#000,.15);
+    z-index: 1010;
   }
   &.chosen-with-drop .chosen-drop {
     left: 0;
   }
-  a{
+  a {
     cursor: pointer;
   }
 
-  .search-choice, .chosen-single{
-    .group-name{
-      margin-right: 4px;
-      overflow: hidden;
-      white-space: nowrap;
-      text-overflow: ellipsis;
-      font-weight: normal;
-      color: #999999;
-      &:after {
-        content: ":";
-        padding-left: 2px;
-        vertical-align: top;
-      }
+  /* include_group_label_in_selected */
+  .group-name {
+    font-weight: bold;
+    margin-right: 4px;
+    &:after {
+      content: ":";
     }
   }
 }
 /* @end */
 
 /* @group Single Chosen */
-.chosen-container-single{
+.chosen-container-single {
   .chosen-single {
-    position: relative;
+    background: $chosen-background;
+    border: 1px solid $chosen-border;
+    border-radius: $chosen-radius;
+    color: $chosen-text;
     display: block;
+    height: $chosen-height;
+    line-height: $chosen-height - 2px;
     overflow: hidden;
-    padding: 0 0 0 8px;
-    height: 25px;
-    border: 1px solid #aaa;
-    border-radius: 5px;
-    background-color: #fff;
-    @include background(linear-gradient(#fff 20%, #f6f6f6 50%, #eee 52%, #f4f4f4 100%));
-    background-clip: padding-box;
-    box-shadow: 0 0 3px #fff inset, 0 1px 1px rgba(#000,.1);
-    color: #444;
+    padding: 0 25px 0 8px;
+    position: relative;
     text-decoration: none;
-    white-space: nowrap;
-    line-height: 24px;
-  }
-  .chosen-default {
-    color: #999;
-  }
-  .chosen-single span {
-    display: block;
-    overflow: hidden;
-    margin-right: 26px;
     text-overflow: ellipsis;
     white-space: nowrap;
-  }
-  .chosen-single-with-deselect span {
-    margin-right: 38px;
-  }
-  .chosen-single abbr {
-    position: absolute;
-    top: 6px;
-    right: 26px;
-    display: block;
-    width: 12px;
-    height: 12px;
-    background: $chosen-sprite -42px 1px no-repeat;
-    font-size: 1px;
-    &:hover {
-      background-position: -42px -10px;
+    &.chosen-default {
+      color: #999;
     }
-  }
-  &.chosen-disabled .chosen-single abbr:hover {
-    background-position: -42px -10px;
-  }
```

Comment:
Lower in this file is an entire section devoted to `.chosen-disabled` support. Moved this line down there.